lib/web/terminal.go

```diff
@@ -404,6 +404,17 @@ func (t *TerminalHandler) handler(ws *websocket.Conn, r *http.Request) {
 	t.log.Debug("Closing websocket stream")
 }
 
+type stderrWriter struct {
+	stream *TerminalStream
+}
+
+func (s stderrWriter) Write(b []byte) (int, error) {
+	if err := s.stream.writeError(string(b)); err != nil {
+		return 0, trace.Wrap(err)
+	}
+	return len(b), nil
+}
```

Comment:
Why is this function slightly different than on master?

Reply:
I will leave it up to you, I think we don't have to backport that previous changes.